### DISCUSSION TITLE
ElasticNet as a default propensity score model

### DISCUSSION THREAD

**Discussion by yejinjkim** [uber/causalml#317] [General]
Hi all, just wondering why ElasticNet is the default propensity score model. If treatment is binary or discrete, then propensity score will be probability to receive the treatment. ElasticNet is a regression model, in which the value can be out of [0,1]. I believe logistic regression would be better for probability? Thank you!

**Comment by jeongyoonlee** [comment]
Thanks for your question, @yejinjkim. Our ElasticNet implementation is different from Scikit-Learn's and it is a classification model. It uses LogisticRegression with ElasticNet regularization (L1 + L2).
